exporters/otprotocol/src/main/java/io/opentelemetry/exporters/otprotocol/TraceProtoUtils.java

```diff
@@ -47,4 +48,17 @@ public static ByteString toProtoTraceId(TraceId traceId) {
     traceId.copyBytesTo(traceIdBytes, 0);
     return ByteString.copyFrom(traceIdBytes);
   }
+
+  /**
+   * Converts a opentelemetry Timestamp into a protobuf Timestamp.
+   *
+   * @param timestamp the opentelemetry Timestamp to convert.
+   * @return the protobuf Timestamp representation.
+   */
+  public static Timestamp toProtoTimestamp(io.opentelemetry.common.Timestamp timestamp) {
```

Comment:
This method doesn't have a unit test. Would be good to add it.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/609